configure

```diff
@@ -1728,6 +1729,9 @@ Optional Features:
                           Unix install. optional INSTALLDIR specifies the
                           installation path. see Mac/README.rst (default is
                           no)
+  --enable-wasm-dynamic-linking
+                          Enable dynamic linking support for WebAssembly
+                          (default is no)
   --enable-shared         enable building a shared Python library (default is
```

Comment:
when using --enable-wasm-dynamic-linking i think --enable-shared should set  -fPIC and -s SIDE_MODULE=1 for libpython.so

Reply:
Thanks! I added a note to the BPO ticket.